### PR TITLE
Register page skeleton

### PR DESCRIPTION
**Pull request by joseph-zou**
Before: 
image

After:
image

I added tailwindcss, updated the pages/Auth/Register.tsx. Created a Register components folder in src/components/RegisterForms. 

Was told we are not implementing 3rd party sign in, however I have all the svgs and can be implemented if needed. 

I still need to do the 3 account type models.

### CODE REVIEW

**Comment by joseph-zou** [comment]
Before : 
image

After :
image

I added tailwind and a couple of files in the components folder. Changed 9 files.

**Comment by joseph-zou** [comment]
image

updated the registration

File: MTM-Frontend/src/pages/Auth/Register.css
Comment (OliviaLiu417) [inline MTM-Frontend/src/pages/Auth/Register.css:1]: If this file is empty maybe consider deleting it?

File: MTM-Frontend/src/pages/Auth/Register.tsx [hunk 1]
```diff
@@ -5,30 +5,61 @@ import { Link, useNavigate } from "react-router-dom";
...
+  name: Yup.string()
+    .matches(/^([A-Za-z]+\s[A-Za-z]+)$/, "First and Last name")
+    .required("First and Last name is required"),
   email: Yup.string()
     .email("Invalid email address")
     .required("Email is required"),
   password: Yup.string()
+    .matches(
+      /^(?=.*[0-9])(?=.*[!@#$%^&*])[A-Za-z0-9!@#$%^&*]{8,}$/,
+      "Invalid password",
```
Comment (OliviaLiu417) [inline MTM-Frontend/src/pages/Auth/Register.tsx:45]: If you are adding password rules make sure it gives the user appropriate error message when they input invalid password (eg. let them know special chars, upper/lowercase, etc)

Comment (joseph-zou) [reply MTM-Frontend/src/pages/Auth/Register.tsx:45]: In the password input, underneath it there's a message for 1 special and 1 number character.

**Review by OliviaLiu417 — APPROVED** [review]

**Pull request merged**